core/src/main/java/org/stellar/anchor/client/CustodialClient.java

```diff
@@ -8,7 +8,7 @@
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
-public class CustodialClientConfig implements ClientConfig {
+public class CustodialClient implements ClientConfig {
```

Comment:
Not to confuse with the API client, The original name `CustodialClientConfig` will be less ambiguous and consistent with the `ClientConfig` naming.

Reply:
I tried to avoid using 'config' in the naming because these two classes represent client objects. They are different from the other `xxxConfig` classes, which primarily describe configuration settings

Reply:
Examples of API clients are `Sep12Client` or `Sep24Client`.